### DISCUSSION TITLE
CONTACT2: which option to select when loadbalance (PWR SHARE) enabled?

### DISCUSSION THREAD

**Discussion by VelocityMaximus** [dingo35/SmartEVSE-3.5#69] [Q&A]
Which 'CONTACT2' option to select when having 'PWR SHARE' enabled on a 3 phase 16A setup?

The option is 'Auto' or 'Solar Off' when loadbalancing is enabled.

When 'Solar Off' is selected, the maximum will be 16A on one phase in my setup and never switches on C2 as far I can figure out.
If I use 'Auto', the description says that the C2 contact will always be on and thus uses 3 phases.

Is my assumption right?

If this is the case, solar charging with loadbalancing enabled does not switch C2 automatic and is feeding 3 phases always without calculating the total solar current on 3 phases.

**Comment by dingo35** [comment]
If you are on current v3.5.1, your options will be "Not Present, Always On, Always Off, Solar Off, Auto" when PWR SHARE = Master. 

Not sure what version you're on; the term "loadbalancing" is ambiguous.

**Comment by VelocityMaximus** [comment]
I have just installed 3.5.1

The manual says: CONTACT2 will be set to ALWAYS_ON when PWR SHARE is enabled.

Which settings do I have to choose for a 3 phase system with 2 EVSE modules in a master/slave config?

**Comment by dingo35** [comment]
Ok that is an old remark, I just removed it from the manual since it is not valid anymore.

The CONTACTOR2 setting is only working on the Master, obviously it will only work on the Master EVSE.
On the slaves that setting is not available, so the slave will always charge on 1 phase if you wired a C2 in there.
